openedx/core/djangoapps/site_configuration/models.py

```diff
@@ -89,15 +89,27 @@ def get_all_orgs(cls):
         for example, to do filtering.
 
         Returns:
-            Configuration value for the given key.
+            A list of all organizations present in site configuration.
         """
         org_filter_set = set()
-        for configuration in cls.objects.filter(values__contains='course_org_filter', enabled=True).all():
+
+        configurations = cls.objects.filter(values__contains='course_org_filter', enabled=True).all()
+        for configuration in configurations:
```

Comment:
This changes seem unnecessary. Is there a reason to assign to the `configurations` variable?